docs/content/direct/combined-status.md

```diff
@@ -8,38 +8,81 @@ It may make sense to expand the latter into a general feature in the future, acc
 
 ### Relation with SQL
 
-To a given workload object, the user can bind one or more named "status combiners". Each status combiner is analogous to an SQL SELECT statement that does the following things.
+To a given workload object, and in the context of a given `Binding` object, the user has bound some `StatusCollector` objects. The meaning of a `StatusCollector` in the context of a (workload object, `Binding` object) pair is analogous to an SQL SELECT statement that does the following things.
 
-1. The SELECT statement has one input table, and that has a row per WEC that the object is prescribed to propagate to.
+1. The SELECT statement has one input table, which has a row per WEC that the workload object is prescribed to propagate to.
 
 1. The SELECT statement can have a WHERE clause that filters out some of the rows.
 
 1. The SELECT statement either does aggregation or does not. In the case of not doing aggregation, the SELECT statement simply has a collection of named expressions defining the columns of its output.
 
 1. In the case of aggregation, the SELECT statement has the following.
 
-   - A `GROUP BY` clause saying how the rows (WECs) are grouped to
-     form the inputs for aggregation, in terms of named
-     expressions. For convenience here, each of these named
-     expressions is implicitly included in the output columns.
+    - An optional `GROUP BY` clause saying how the rows (WECs) are
+      grouped to form the inputs for aggregation, in terms of named
+      expressions. For convenience here, each of these named
+      expressions is implicitly included in the output columns.
 
-   - A collection of named expressions using aggregation functions to define
-     additional output columns.
+    - A collection of named expressions using aggregation functions to define
+      additional output columns.
 
 1. The SELECT statement has a LIMIT on the number of rows that it will yield.
 
+#### Relation with SQL, more precisely
+
+For a given workload object, `Binding`, and `StatusCollector`: start with a table named `PerWEC`. This table has one primary key column and it holds the name of the WEC that the reported state is from. The dependent columns hold the workload object content from the WDS, the workload object content returned from the WEC, and the augmentations.
+
+There are three forms of `StatusCollector`, equivalent to three forms of SQL statement.
+
+##### Plain selection
+
+When the `StatusCollector` has selection but no "GROUOP BY" and no aggregation, this is equivalent to the following form of SELECT statement.
+
+```sql
+SELECT <selected columns>
+FROM PerWEC WHERE <filter condition>
+LIMIT <limit>
+```
+
+##### Aggregation without GROUP BY
+
+When there is aggregation but no plain selection and _no_ "GROUP BY", this is equivalent to the following form of SELECT statement.
+
+```sql
+SELECT <aggregation columns>
```

Comment:
I think it would be helpful to expand on what aggregation columns are. Lines 42-43 give the idea, but it would be helpful to show the full equivalent SQL example in which a function is applied, at least once.

Reply:
I have updated #2325